redesign/src/Tests/Common/BitwiseTicTacToe.jl

```diff
@@ -54,6 +54,13 @@ function BatchedEnvs.act(env::BitwiseTicTacToeEnv, pos)
     return newenv, (; reward, switched=true)
 end
 
+function BatchedEnvs.act(env::BitwiseTicTacToeEnv, pos_list::AbstractArray)
+    for pos in pos_list
+        env, vec = BatchedEnvs.act(env, pos)
+    end
+    return env, vec # TODO: returning `vec` is surely not the best thing to do
```

Comment:
Note: There is still a # TODO in it, but it will be fixed in a later PR (and it does not matter for how the environment is currently used.